pkg/apis/scorecard/v1alpha2/formatter.go

```diff
@@ -77,6 +77,12 @@ func (s ScorecardOutput) MarshalText() (string, error) {
 		for _, err := range result.Errors {
 			sb.WriteString(fmt.Sprintf("\t\t%s\n", err))
 		}
+
+		if result.Log != "" {
+			sb.WriteString(fmt.Sprintf("\tLog:\n"))
+			sb.WriteString(fmt.Sprintf("\t\t%s\n", result.Log))
+		}
```

Comment:
Do we want to condition the log output on other failures in the scorecard test. If the test passes, maybe it makes sense to skip the log output?

Reply:
I was wondering about that at one point and figured the test writer might still want to include log output in a passing test if they choose to.  but I'm pretty flexible if you think this might never be useful to a test developer, I was thinking more of future custom tests as the place where this might be useful.

Reply:
I've removed the CSVValidator test since it really doesn't buy anything since the csv is being validated prior to test execution.

Reply:
Yeah, this is another refactor thing probably. My thought would be to do something like the Go test system where tests can use `t.Log` to log stuff. Then when the test is actually run, I think the logic on outputting that log is when the test fails and/or the test was run with `-v`. I would envision something similar for this.

That way in the happy case, the output is not spammed with a bunch of extraneous logs.